### PR TITLE
IN-D Insurance (ICD10/CPT autocoding)

### PR DESCRIPTION
**Pull request by gayathri35**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

- [ ] I attest that the connector works and I verified by deploying and testing all the operations. 
- [ ] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [ ] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [ ] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [ ] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json [hunk 1]
```diff
@@ -437,19 +450,23 @@
...
                       "cpt_code": {
                         "type": "string",
-                        "description": "cpt_code"
+                        "description": "list of cpt codes",
+                        "title": "cpt code"
                       },
                       "cpt_desc": {
                         "type": "string",
-                        "description": "cpt_desc"
+                        "description": "cpt_desc",
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json:468]: @gayathri35 All the description values of properties should be free from grammatical and spell errors. please provide valid description values for all the properties.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json [hunk 2]
```diff
@@ -437,19 +450,23 @@
...
                       "cpt_code": {
                         "type": "string",
-                        "description": "cpt_code"
+                        "description": "list of cpt codes",
+                        "title": "cpt code"
                       },
                       "cpt_desc": {
                         "type": "string",
-                        "description": "cpt_desc"
+                        "description": "cpt_code_description",
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json:468]: @gayathri35 All the description values of properties should be free from grammatical and spell errors. please provide valid description values for all the properties.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json (hunk 2 above)
Comment (gayathri35) [reply certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json:468]: Hi@sriyen-msft,
Sorry for the mistake, I have corrected it please do check once,
 },
                      "cpt_description": {
                        "type": "string",
                        "description": "cpt_code_description",
                        "title": "cpt code description"
                      },

File: certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "swagger": "2.0",
   "info": {
-    "title": "IN-D Insurance (ICD10/CPT)",
+    "title": "IN-D ICD10/CPT autocoding",
```
[the remark this answers is not in the document]
Comment (gayathri35) [reply certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json:4]: Hi@sriyen-msft,
I have Corrected the Title name as "IN-D Insurance (ICD10 & CPT)" please check once.
{
  "swagger": "2.0",
  "info": {
    "title": "IN-D Insurance ICD10 & CPT",

File: certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json (hunk 2 above)
Comment (sriyen-msft) [reply certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json:468]: @gayathri35 Title and Description values are different. I have provided feedback for description values of all the properties in the swagger. 

Ex.: Line number 468: `"description": "cpt_code_description"` - This is not allowed it has grammatical and spell errors. Please fix all other description values of properties in the swagger.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json [hunk 4]
```diff
@@ -437,19 +450,23 @@
...
                         "type": "string",
-                        "description": "cpt_code"
+                        "description": "list of cpt codes",
+                        "title": "cpt code"
                       },
                       "cpt_desc": {
                         "type": "string",
-                        "description": "cpt_desc"
+                        "description": "cpt_code_description",
+                        "title": "cpt desc"
```
Comment (gayathri35) [inline certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json:469]: Hi@sriyen-msft,
Sorry for the mistake, I have corrected it please do check once,
},
"cpt_desc": {
"type": "string",
"description": "cpt_code_description",
"title": "cpt desc"
},

File: certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/README.md [hunk 5]
```diff
@@ -1,24 +1,23 @@
-# IN-D Insurance (ICD10/CPT)
+# IN-D Insurance (ICD10/CPT autocoding)
```
Comment (sriyen-msft) [inline certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/README.md:1]: @gayathri35 The title should be changed everywhere in the connector artifacts.

**Comment by sriyen-msft** [comment]
@gayathri35 I'm just checking in to see how things are going with updating your connector artifacts as per the given feedback. Please feel free to reach out with any questions you may have, Thanks.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/README.md (hunk 5 above)
Comment (gayathri35) [reply certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/README.md]: Hi Srikanth,
I have changed the title: IN-D Insurance (ICD10 & CPT) everywhere in the connector artifacts. Please check once. Sorry for the inconvenience.

File: certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json (hunk 2 above)
Comment (sriyen-msft) [reply certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json:468]: @gayathri35 Please resolve this comment to get your PR approved and merged to move forward in the certificaiton.

**Review by sriyen-msft — CHANGES_REQUESTED** [review]

File: certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json (hunk 2 above)
Comment (gayathri35) [reply certified-connectors/IN-D Insurance (ICD10 & CPT autocoding)/apiDefinition.swagger.json]: Hi Srikanth,
I have changed the description comment. Please do check once, Sorry for the mistake.
"cpt_desc": {
                        "type": "string",
                        "description": "cpt_description",
                        "title": "cpt desc"
                      },

**Review by sriyen-msft — APPROVED** [review]

**Pull request merged**